src/index.ts

```diff
@@ -266,7 +266,7 @@ export function main(argString = "") {
   cliExecute("ccs garbo");
 
   // FIXME: Dynamically figure out pointer ring approach.
-  withStash($items`haiku katana, repaid diaper`, () => {
+  withStash($items`haiku katana, repaid diaper, moveable feast`, () => {
```

Comment:
I don't want to take out the moveable feast for the entire duration of farming if we can avoid it. Can you change withStash to just not pull if it's not available?

Reply:
Addressed as of 15879b0